--- configure	2015-02-02 21:42:55.000000000 +0100
+++ configure	2015-02-14 22:50:43.786625024 +0100
@@ -8262,7 +8262,7 @@
 else	READLINE_ORDER="-ledit -lreadline"
 fi
 for pgac_rllib in $READLINE_ORDER ; do
-  for pgac_lib in "" " -ltermcap" " -lncurses" " -lcurses" ; do
+  for pgac_lib in "" " -lncurses" " -ltermcap" " -lcurses" ; do
     LIBS="${pgac_rllib}${pgac_lib} $pgac_save_LIBS"
     cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
