diff --git Code/PgSQL/rdkit/adapter.cpp Code/PgSQL/rdkit/adapter.cpp
index 877e9cc..c733bd5 100644
--- Code/PgSQL/rdkit/adapter.cpp
+++ Code/PgSQL/rdkit/adapter.cpp
@@ -50,10 +50,10 @@
 #include <DataStructs/SparseIntVect.h>
 #include <boost/integer_traits.hpp>
 #ifdef BUILD_INCHI_SUPPORT
-#include <INCHI-API/inchi.h>
+#include <GraphMol/inchi.h>
 #endif
 #ifdef BUILD_AVALON_SUPPORT
-#include <AvalonTools/AvalonTools.h>
+#include <GraphMol/AvalonTools.h>
 #endif
 #include "rdkit.h"
 #include <GraphMol/ChemReactions/ReactionFingerprints.h>
