diff --git Code/PgSQL/rdkit/CMakeLists.txt Code/PgSQL/rdkit/CMakeLists.txt
index fb062165..d565e32d 100644
--- Code/PgSQL/rdkit/CMakeLists.txt
+++ Code/PgSQL/rdkit/CMakeLists.txt
@@ -1,5 +1,19 @@
 cmake_minimum_required(VERSION 2.8.8)
 
+project(RDKitPostgreSQL)
+enable_testing()
+
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Modules/")
+
+find_package(RDKit REQUIRED)
+include_directories(${RDKit_INCLUDE_DIRS})
+
+# Boost and Eigen libraries that are required at link time
+find_package(Boost 1.56.0 COMPONENTS serialization thread)
+find_package(Eigen3)
+find_package(Cairo)
+
 if(APPLE)
   set (EXTENSION_PREFIX "")
   set (EXTENSION_SUFFIX ".so")
@@ -27,14 +41,14 @@ if(WIN32)
     include_directories(${PostgreSQL_INCLUDE_DIR}/server/port/win32_msvc)
   endif(MSVC)
 endif(WIN32)
-if(RDK_BUILD_AVALON_SUPPORT)
-  include_directories(${CMAKE_SOURCE_DIR}/External)
+#if(RDK_BUILD_AVALON_SUPPORT)
+#  include_directories(${CMAKE_SOURCE_DIR}/External)
   add_definitions(-DBUILD_AVALON_SUPPORT)
-endif(RDK_BUILD_AVALON_SUPPORT)
-if(RDK_BUILD_INCHI_SUPPORT)
-  include_directories(${CMAKE_SOURCE_DIR}/External)
+#endif(RDK_BUILD_AVALON_SUPPORT)
+#if(RDK_BUILD_INCHI_SUPPORT)
+#  include_directories(${CMAKE_SOURCE_DIR}/External)
   add_definitions(-DBUILD_INCHI_SUPPORT)
-endif(RDK_BUILD_INCHI_SUPPORT)
+#endif(RDK_BUILD_INCHI_SUPPORT)
 add_definitions(-DRDKITVER="007300")
 if(NOT DEFINED PostgreSQL_ROOT)
   if(DEFINED ENV{PostgreSQL_ROOT})
@@ -96,14 +110,14 @@ if(RDK_BUILD_AVALON_SUPPORT)
   set(pgRDKitLibList "${pgRDKitLibList}AvalonLib;avalon_clib;")
 endif(RDK_BUILD_AVALON_SUPPORT)
 if(RDK_BUILD_INCHI_SUPPORT)
-  if(NOT DEFINED INCHI_LIBRARIES)
-    set(CMAKE_MODULE_PATH
-        ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Modules/")
-    if(NOT DEFINED RDKit_RELEASENAME)
-      include(RDKitUtils)
-    endif(NOT DEFINED RDKit_RELEASENAME)
-    find_package(Inchi)
-  endif(NOT DEFINED INCHI_LIBRARIES)
+  #if(NOT DEFINED INCHI_LIBRARIES)
+  #  set(CMAKE_MODULE_PATH
+  #      ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Modules/")
+  #  if(NOT DEFINED RDKit_RELEASENAME)
+  #    include(RDKitUtils)
+  #  endif(NOT DEFINED RDKit_RELEASENAME)
+  #  find_package(Inchi)
+  #endif(NOT DEFINED INCHI_LIBRARIES)
   set(inchiRegress "inchi")
   set(pgRDKitLibList "${pgRDKitLibList}RDInchiLib;${INCHI_LIBRARIES};")
 endif(RDK_BUILD_INCHI_SUPPORT)
