diff --git Code/PgSQL/rdkit/CMakeLists.txt Code/PgSQL/rdkit/CMakeLists.txt
index 2ef36c7..b46a119 100644
--- Code/PgSQL/rdkit/CMakeLists.txt
+++ Code/PgSQL/rdkit/CMakeLists.txt
@@ -1,5 +1,13 @@
 cmake_minimum_required(VERSION 2.8.8)
 
+project(RDKitPostgreSQL)
+enable_testing()
+
+find_package(RDKit REQUIRED)
+include_directories(${RDKit_INCLUDE_DIRS})
+
+find_package(PostgreSQL REQUIRED)
+
 if(APPLE)
   set (EXTENSION_PREFIX "")
   set (EXTENSION_SUFFIX ".so")
