--- Code/PgSQL/rdkit/CMakeLists.txt	2019-12-26 15:35:18.050820200 +0100
+++ Code/PgSQL/rdkit/CMakeLists.txt	2019-12-26 15:57:33.296451700 +0100
@@ -15,8 +15,10 @@
 endif(RDK_OPTIMIZE_NATIVE)
 
 set(EXTENSION rdkit)
+set(Boost_USE_STATIC_LIBS OFF)
+set(BUILD_SHARED_LIBS ON)
 if(NOT DEFINED Boost_INCLUDE_DIRS)
-  find_package(Boost 1.56.0 REQUIRED)
+  find_package(Boost 1.56.0 COMPONENTS iostreams REQUIRED)
 endif(NOT DEFINED Boost_INCLUDE_DIRS)
 include_directories(${Boost_INCLUDE_DIRS})
 message("postgres: ${PostgreSQL_INCLUDE_DIRS}")
@@ -82,7 +84,7 @@
   add_library(${EXTENSION} SHARED
               adapter.cpp bfp_op.c cache.c guc.c low_gist.c mol_op.c
               rdkit_gist.c bfp_gist.c bfp_gin.c bitstring.c rdkit_io.c rxn_op.c sfp_op.c)
-  target_link_libraries(${EXTENSION} ${PostgreSQL_LIBRARIES})
+  target_link_libraries(${EXTENSION} Boost::iostreams ${PostgreSQL_LIBRARIES})
   if(WIN32)
     target_link_libraries(${EXTENSION} postgres)
   endif(WIN32)
