diff --git Code/PgSQL/rdkit/adapter.cpp Code/PgSQL/rdkit/adapter.cpp
index 67bd996..1bcf1c1 100644
--- Code/PgSQL/rdkit/adapter.cpp
+++ Code/PgSQL/rdkit/adapter.cpp
@@ -59,10 +59,10 @@
 #include <RDGeneral/BoostEndInclude.h>
 
 #ifdef BUILD_INCHI_SUPPORT
-#include <INCHI-API/inchi.h>
+#include <GraphMol/inchi.h>
 #endif
 #ifdef BUILD_AVALON_SUPPORT
-#include <AvalonTools/AvalonTools.h>
+#include <GraphMol/AvalonTools.h>
 #endif
 #include <GraphMol/ChemReactions/ReactionFingerprints.h>
 #include <GraphMol/ChemReactions/ReactionUtils.h>
